DiscordChatExporter.Cli.dockerfile

```diff
@@ -12,7 +12,9 @@ RUN dotnet publish DiscordChatExporter.Cli -c Release -o ./publish
 # Run
 FROM mcr.microsoft.com/dotnet/runtime:6.0 AS run
 
-COPY --from=build ./publish ./
+COPY --from=build ./publish /opt/dce
```

Comment:
I'm not well-versed in nix systems, does copying it to `/opt/dce` make it available for the user `dce`? Could you please explain the solution to me like I'm 5 years old? 🙂

Reply:
Actually, same reason as above. I think it is better practice to put the application files in a separate folder instead of the root.

It's like you're putting the application files straight on C:\ :)